content/rc/databases/high-availability.md

```diff
@@ -9,19 +9,19 @@ categories: ["RC"]
 aliases: 
 ---
 
-Database replication helps ensure high availability. When replication is enabled, your dataset is duplicated to create a replica, which stays in sync with the primary dataset. Replication allows for automatic failover and greater fault tolerance. This can prevent data loss in the event of a hardware, zone, or regional failure. 
+Database replication helps ensure high availability. When replication is enabled, your dataset is duplicated to create a replica, which stays in sync with the primary dataset. Replication allows for automatic failover and greater fault tolerance. This can prevent data loss in the event of a hardware, or zone failure. 
 
 Redis Enterprise Cloud supports three levels of replication:
 
 - _No replication_ means that you will have a single copy of your database.
-- _Single-zone replication_ means that your database will have a primary and a replica located in the same cloud region. If anything happens to the primary, the replica takes over and becomes the new primary.
-- _Multi-zone replication_ means that the primary and its replicas are stored in different regions. This means that your database can remain online even if an entire region becomes unavailable.
+- _Single-zone replication_ means that your database will have a primary and a replica located in the same zone of a cloud region. 
+- _Multi-zone replication_ means that the primary and its replicas are stored in different zones of a cloud region. This means that your database can remain online even if an entire zone becomes unavailable.
 
 Your replication options depend on which subscription plan you're using:
 
 - _Free_ plans do not support replication.
-- _Fixed_ plans allow you to choose between no replication or single-zone replication.
-- _Flexible_ plans enable single-zone replication by default and allow you to upgrade to multi-zone replication. 
+- _Fixed_ plans allow you to choose between no replication, single-zone replication or multi-zone replication during the subscription creation. There is no option to upgrade or downgrade between signle-zone to multi-zone subscirption. There is no option to turn-off replication on the multi-zone fixed subscirption.
```

Comment:
```suggestion
- _Fixed_ plans let you choose between no replication, single-zone replication, or multi-zone replication when creating a subscription.  

    Once a subscription has been created, you cannot change replication or zone settings.  (To make such changes, create a new subscription and then migrate your data to it.)
```

Reply:
This changes the meaning: "Once a subscription has been created, you cannot change replication" 
This page doesn't talk about cache/standard/multi-az subscription, so I don't think we should add details that we can migrate from no HA plan to Single HA plan. I do think we should point out that there is not way to upgrade/downgrade between the single zone/multi-zone subscriptions. 

How about: Once a subscription has been created, you cannot upgrade/downgrade from single to multi-zone subscription or change zone settings.  (To make such changes, create a new subscription and then migrate your data to it using replica of or backup and restore.)

Reply:
Updated to the language we discussed off-line.